### PR TITLE
Adding namespace or label selector error; delete_count option

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Adding in fix to have namespace match all with a specific label selector OR label selector set to "" with a set namespace 

Also adding in option to set delete_count (number of namespaces matching label or name to delete) and runs (rerunning the whole scenario with post action in between)

### Fixes
https://github.com/cloud-bulldozer/kraken/issues/191

### CODE REVIEW

File: kraken/namespace_actions/common_namespace_functions.py [hunk 1]
```diff
@@ -19,48 +19,61 @@ def run(scenarios_list, config, wait_duration, failed_post_scenarios, kubeconfig
...
-                        )
-                        logging.info("Namespace action error: " + str(e))
-                        sys.exit(1)
-                    namespaces.remove(selected_namespace)
-                    logging.info("Waiting %s seconds between namespace deletions" % str(run_sleep))
-                    time.sleep(run_sleep)
+                    logging.info("run count" + str(i))
+                    namespaces = kubecli.check_namespaces([scenario_namespace], scenario_label)
+                    for j in range(delete_count):
+                        logging.info("delete count" + str(j))
```
Comment (chaitanyaenr) [inline kraken/namespace_actions/common_namespace_functions.py:41]: Maybe change the format to "delete count: " to improve readability? Thoughts?

**Review by chaitanyaenr — COMMENTED** [review]
Minor nit

File: docs/namespace_scenarios.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ Configuration Options:
 
 **label_selector:** label on the namespace you want to delete
```
Comment (chaitanyaenr) [inline docs/namespace_scenarios.md:11]: Maybe mention about just setting either label_selector or namespace here? Thoughts?

File: kraken/namespace_actions/common_namespace_functions.py (hunk 1 above)
Comment (paigerube14) [reply kraken/namespace_actions/common_namespace_functions.py:41]: didn't mean to keep that in there, thoughts on just deleting that line and line 38?

Comment (chaitanyaenr) [reply kraken/namespace_actions/common_namespace_functions.py]: Sure, should be print the config to help understand what values were passed?

Comment (paigerube14) [reply kraken/namespace_actions/common_namespace_functions.py]: Updated

File: docs/namespace_scenarios.md [hunk 3]
```diff
@@ -6,7 +6,9 @@ Configuration Options:
 
 **action:** default is `delete`
 
-**namespace:** specific namespace or regex style namespace of what you want to delete, gets all namespaces if not specified; set to "^.*$" if you want to use the label_selector field
+**namespace:** specific namespace or regex style namespace of what you want to delete, gets all namespaces if not specified; set to "" if you want to use the label_selector field
+
+Set to '^.*$' and label to "" to randomly select any namespace in your cluster
```
Comment (chaitanyaenr) [inline docs/namespace_scenarios.md:11]: s/label/label_selector

Comment (paigerube14) [reply docs/namespace_scenarios.md]: done

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**